CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 Fix: modified engine version >=6 in packages.json
 Fix: update logops dependency from 1.0.8 to 2.1.0
+Add: config.http.timeout (and associated enviroment variable IOTA_HTTP_TIMEOUT)

```

Comment:
Please add at the end of the line also the issue number being solved, i.e:

```
 Add: config.http.timeout (and associated enviroment variable IOTA_HTTP_TIMEOUT) (#97)
```

Reply:
Fixed in 4a42f8f